dts/riscv/virt.dtsi

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2020 Cobham Gaisler AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file is based on:
+ *   qemu-system-riscv32 -machine virt,dumpdtb=virt.dtb -smp 8 -m 256
+ *   dtc virt.dtb > virt.dtsi
+ */
+
+/dts-v1/;
+
+/ {
+	#address-cells = < 0x01 >;
+	#size-cells = < 0x01 >;
+	compatible = "riscv-virtio";
```

Comment:
The other two compatibles that the check we're going to discard have precedence in Linux, but this one does not.

There is a `virtio` manufacturer for things like `virtio,mmio` but I don't think `virtio,riscv` makes sense.

But there is a `riscv` manufacturer for things like `riscv,isa`.

Maybe this should be `riscv,virtio` instead of creating a new non-namespaced compatible?  @galak

Reply:
Good point. The general idea was to keep the DTS as similar as possible to the DTS dumped by QEMU. That would simplify updates in the future if we for example want to add QEMU peripherals.

I am fine with changing this compatible line.
As an alternative solution, would it be acceptable to just remove this line from the file?

Reply:
No, you need a compatible.  I see that this compatible is what qemu emits, so presumably it needs to be this string.